Makefile

```diff
@@ -1,19 +1,23 @@
-ddown = docker-compose down
-
-install: build migrate
+# General =====================================================================
+install: build migrate import_initial_data
 
 build:
 	docker-compose build
 
 up:
 	docker-compose up
 
+ddown = docker-compose down --remove-orphans
 down:
-	$(ddown) --remove-orphans
+	$(ddown)
 
 manage:
 	docker-compose run --rm web python manage.py $(command)
 
+import_initial_data:
```

Comment:
I'd add add migrate here probs

Reply:
why, import means import, not migrate and import